### PR TITLE
Adds lock screen icon URL field to policy table.

### LINKED ISSUE #3: Custom Lock Screen Icon Defined by the OEM
We want the smartphone to display a logo in the lock screen that is defined by the OEM and based on the connected vehicle.  The current proposed solution (POC) for SDL Core is [on GENIVI](http://git.projects.genivi.org/?p=smartdevicelink.git;a=shortlog;h=refs/heads/dev/lock_screen_icon_url)

The solution sends an OnSystemRequest notification with a URL that points to the logo for the application to download, cache, and display based on the vehicle type returned in the register app interface response.

### PR DESCRIPTION
**Pull request by lea255ace**
Methods added to extract URL from policy table and send to mobile.

resolves #3

### CODE REVIEW

File: src/appMain/sdl_preloaded_pt.json [hunk 1]
```diff
@@ -14,7 +14,10 @@
 			"endpoints": {
 				"0x07": {
 					"default": ["http://policies.telematics.ford.com/api/policies"]
-				}
+				},
+			"lock_screen_icon_url": {
+			        "default": ["http://www.livioconnect.com/wp-content/uploads/2012/03/icon-Livio-Connect.png"]
```
Comment (justinjdickow) [inline src/appMain/sdl_preloaded_pt.json]: Let's change this to the smartdevicelink icon

Comment (ssmereka) [inline src/appMain/sdl_preloaded_pt.json]: SmartDeviceLink icon on imgur:  http://i.imgur.com/QwZ9uKG.png

**Comment by lea255ace** [comment]
Having issues producing the correct behavior, putting this PR on hold.

**Comment by lea255ace** [comment]
Issues resolved

**Pull request merged**